lib/install.go

```diff
@@ -25,17 +25,17 @@ var (
 	installLocation = "/tmp"
 )
 
-// initialize : removes existing symlink to terraform binary// I Don't think this is needed
-func initialize() {
+// initialize : removes existing symlink to terraform binary based on provided binPath
+func initialize(binPath string) {
 
 	/* Step 1 */
 	/* initilize default binary path for terraform */
 	/* assumes that terraform is installed here */
 	/* we will find the terraform path instalation later and replace this variable with the correct installed bin path */
-	installedBinPath := "/usr/local/bin/terraform"
+	installedBinPath := binPath + "terraform"
```

Comment:
Should we ensure the `binPath` has trailing slash?

Reply:
Good call

Reply:
We actually don't need this line at all because `/terraform` (or whatever a user desires as executable name) is already expected with the binPath flag.

Reply:
Ah, so it's inherited from `custBinPath` which defaults to `defaultBin` which is `/usr/local/bin/terraform` via `main.go` (if I didn't miss something though 😛)